Carousels/3D Carousel_01/README.md

```diff
@@ -0,0 +1,6 @@
+# 3D Carousel
+<h3> Tech Stack Used <img src = "https://media2.giphy.com/media/QssGEmpkyEOhBCb7e1/giphy.gif?cid=ecf05e47a0n3gi1bfqntqmob8g9aid1oyj2wr3ds3mg700bl&rid=giphy.gif" width = 32px> </h3>
+<p align="left"> <a href="https://www.w3.org/html/" target="_blank"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/html5/html5-original-wordmark.svg" alt="html5" width="40" height="40"/> </a> <a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/css3/css3-original-wordmark.svg" alt="css3" width="40" height="40"/> </a>   </p>
+<hr>
+
+<h2><a href="https://streamable.com/9sb97s">Preview</a></h2>
```

Comment:
Preview URL seems to be broken. Also if you want to show a gif in README you can simply use this syntax

<h2>"![Alt text](url)"</h2>

![Sample gif](https://media.giphy.com/media/XgSCVdHUgGygoUHFGv/giphy.gif)

Reply:
URL is working for me 😅. After clicking on preview you will be redirected to https://streamable.com/9sb97s this url. As gif size was too large it was not loading in readme so I added link to that video 😅

Reply:
@Ovenoboyo what should I do now?

Reply:
Add the gif to your folder then Link it

Reply:
Okay

Reply:
> URL is working for me 😅. After clicking on preview you will be redirected to https://streamable.com/9sb97s this url. As gif size was too large it was not loading in readme so I added link to that video 😅

My bad, streamable was being blocked by my ISP

Reply:
@Abirpal202049   

<img src="https://i.ibb.co/pWns12V/Screenshot-1498.png" alt="Screenshot-1498" border="0">

I got this. I also tried compressing gif with veed.io but its not allowing me to upload large size gif 😓

Reply:
@suyogkokaje if Large Size is not Supported Use some Online GIF Shrinker to Reduce the Size and then try attaching it.

Reply:
@saikatmu31 I tried veed.io but its not allowing me to upload this karte file for compression 😓

Reply:
large* file